data/org.freedesktop.portal.RemoteDesktop.xml

```diff
@@ -38,8 +38,8 @@
 
         A remote desktop session may only be started and stopped with this interface,
         but you can use the #org.freedesktop.portal.Session object created with this
-        method together with certain methods on the #org.freedesktop.portal.ScreenCast
-        interface. Specifically, you can call
+        method together with certain methods on the #org.freedesktop.portal.ScreenCast and
+        #org.freedesktop.portal.Clipboard interfaces. Specifically, you can call
         org.freedesktop.portal.ScreenCast.SelectSources() to also get screen content,
```

Comment:
Also explicitly mention that you can call `#org.freedesktop.portal.Clipboard.RequestClipboard()` to request clipboard integration.